umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/azure-devops.md

```diff
@@ -1,178 +1,222 @@
-{% endhint %}
 
-#### Pipeline Stage: Build, Test, and Package
+It is recommended to handle the `API Key` as a secret. This can be done by ticking the "Keep this value secret" checkbox.  
 
-In this stage, the Umbraco CMS project is built and tested. While this step is optional, it provides significant value. Especially if you're running custom frontend builds with different npm versions for both the backoffice and website views.
+<figure><img src="../../../../.gitbook/assets/azdevops-secret-variable.png" alt=""><figcaption><p>Make the variable secret in Azure DevOps</p></figcaption></figure>
 
-Automated tests can be run to ensure everything is functioning as expected. Additionally, manual verification steps can be included, such as deploying to a local server for QA validation.
+{% endhint %}
 
-The deployment artifact consists of source files to maintain consistency with Umbraco Cloud's existing Git-based deployment flow. Only zip-archived files are currently supported, and the folder structure must align with a standard Umbraco Cloud project.
 
 {% hint style="info" %}
-As part of the sample scripts, we have included two text files to help you pack your source files. You should only use one of the approaches
-
-* the `.zipignore` file specifies which files and folders to exclude in the zipped archive, ensuring that only the necessary files are packaged, without too much extra configuration. (Recommended approach)
-* the `project-files-to-zip-list` file specifies which files and folders to include in the zipped archive, ensuring that only the necessary files are packaged. (If you change the project structure you will need to edit this file)
+You can customize the names for the variables as you like, however you then need to rename the affected variables in `azure-release-pipeline.yaml`.
 
-The Azure pipeline sample has examples of how to use both of them.
 {% endhint %}
 
-#### Pipeline Stage: Deploy
+When you click on "Save and Run" your first deployment will be triggered. Which means that Azure DevOps is set up with all the needed information to be able to deploy your Cloud project back to Umbraco Cloud.
 
-This stage is responsible for creating a new deployment and ensuring that it can be initiated. For more details, refer to the "Create Deployment API" section.
+### Optional: Run a deployment to confirm
 
-The deployment artifact, a zip file containing the source files, is uploaded at this stage. For more information, see the "Upload Zip Source File" section.
+With everything set up, you may want to confirm that Umbraco Cloud reflects the changes you are sending via your pipeline.
 
-The deployment is then initiated. For further details, consult the "Start Deployment" section.
+While working on your project locally, add a new Document type.
+- Commit the change to `main` and push to your repository.
```

Comment:
shouldn´t this be"- Commit the change to `main` (or `master` if you did not change the branch name) in Branch and push to your repository."